docs/tldr/pages/linux/tmt.md

```diff
@@ -0,0 +1,36 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests. See also: `tmt-run`, `tmt-try`.
+> More information: <https://tmt.readthedocs.io>.
+
+- View documentation for `tmt run`:
+
+`tldr tmt-run`
+
+- View documentation for `tmt try`:
+
+`tldr tmt-try`
+
+- Make project tests manageable by `tmt`:
+
+`tmt init`
+
+- Create a new test with optional arguments like template and link:
+
+`tmt create --template beakerlib --link verifies:issue#1234`
+
+- List available tests, plans, or stories:
+
+`tmt <test|plan|story> ls [<pattern>]`
+
+- Validate metadata:
+
+`tmt lint`
+
+- Docs, test & implementation coverage for selected stories:
+
+`tmt story coverage [<pattern>]`
+
+- Use filter:
+
+`tmt tests ls --filter tag:foo --filter tier:0`
```

Comment:
Good point

Reply:
argh, forgot about adding context in the main page. How would you guys want me to add it? In the `tests ls` example? Maybe one context and one filter?  I've added `--help` as it was mentioned in the style doc, but it also said it's not mandatory, just that --help and --version should be at the end.  
I think it's good to have it there, but if you want to replace it with something...

Reply:
I think `tests show` is most likely to show the affects of an `adjust`. Maybe document it as

> - Show the details of a test (adjusted to the specific context with which it would run)

As for which to replace, `tmt story coverage` seems like quite an advanced feature that initial users would not encounter. But also `tmt --help` feels hand-holdy particularly when it overlaps with the plain `tmt` documented above.

Reply:
Yeah, I would include something like `tmt --context arch=aarch64 test show`

Reply:
Done in 40dba99f.